app/src/main/java/com/kickstarter/libs/utils/extensions/StringExt.kt

```diff
@@ -0,0 +1,56 @@
+package com.kickstarter.libs.utils.extensions
+
+import android.util.Patterns
+import java.util.*
```

Comment:
why do you need here an import from Java?

Reply:
We're using it for the `Locale` java file. Thanks for catching this, I am going to update this to import this specific class